tools/driver/swift-format.py

```diff
@@ -0,0 +1,93 @@
+# This file is a minimal swift-format vim-integration.  To install:
```

Comment:
Historically we've avoided including editor support files in the repo for anything besides syntax coloring, and we *definitely* aren't calling them supported (as your "file bugs at bugs.swift.org" implies). I think you'd have to ask on swift-dev whether that policy has changed.

(And if it has, this belongs in utils/ rather than tools/driver/, and with "vim" in the name.)